quiz-app/src/components/app.js

```diff
@@ -0,0 +1,182 @@
+// import './App.css';
+import { TextField, Grid, FormControl, FormLabel, FormControlLabel, Button, Checkbox } from '@mui/material'
```

Comment:
If this file contains a component (which is the case), the filename should start with a capital. app.js -> App.js